pkg/version/version_test.go

```diff
@@ -16,6 +16,8 @@ package version
 
 import "testing"
 
+const commitSha = "abcdef"
```

Comment:
Can you revert changes to  this file? We since removed this linter.

Reply:
Sure. PTAL